### PR TITLE
Add tombstone kafka messages when deleting artifacts/versions.

### PR DESCRIPTION
**Pull request by EricWittmann**

### CODE REVIEW

File: storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/sql/KafkaSqlSink.java [hunk 1]
```diff
@@ -58,6 +58,13 @@
      * @param record
      */
     public void processMessage(ConsumerRecord<MessageKey, MessageValue> record) {
+        // If the value is null, then this is a tombstone (or unrecognized) message and should not 
+        // be processed.
+        if (record.value() == null) {
+            log.info("Discarded a tombstone message with key: {}", record.key());
```
Comment (famartinrh) [inline storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/sql/KafkaSqlSink.java:64]: what about logging some info from the MessageKey?

Comment (EricWittmann) [reply storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/sql/KafkaSqlSink.java:64]: Oh right, I intended to implement `toString()` on the message key classes.  Will do that.

**Review by famartinrh — APPROVED** [review]
LGTM

File: storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/sql/KafkaSqlSink.java (hunk 1 above)
Comment (EricWittmann) [reply storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/sql/KafkaSqlSink.java:64]: Updated to give each key a unique `toString()`.

**Pull request merged**